libbeat/processors/ratelimit/rate_limit.go

```diff
@@ -87,6 +110,7 @@ func (p *rateLimit) Run(event *beat.Event) (*beat.Event, error) {
 	}
 
 	p.logger.Debugf("event [%v] dropped by rate_limit processor", event)
```

Comment:
Add the id to this debug message so it is easier to correlate with the monitoring counter?

Reply:
Yep, good catch. Will fix.

Reply:
Actually the ID is already included by this line: https://github.com/elastic/beats/pull/23330/files#diff-67f3af920cbdfab0fed6f4a5f8077b14cd27cf91fdf20f53c3e02cac8e0a8b70R82. This is the same thing the `dns` processor is also doing for its logger: https://github.com/elastic/beats/blob/f5fd81729fc1dfab4e12a138893f9aec73921c75/libbeat/processors/dns/dns.go#L63


For example, this is what the log looks like (notice the `instance_id` at the end):

```
2021-01-13T11:20:55.263-0800    DEBUG   [processor.rate_limit]  ratelimit/rate_limit.go:112     event [&{2021-01-13 11:20:55.263477 -0800 PST m=+6.116778961 {} {"agent":{"ephemeral_id":"3b27d985-65b6-4591-9a21-5d53ca6a6721","id":"d8740c81-a061-490a-8c17-340ff37a95b3","name":"Shaunaks-MacBook-Pro-Work.local","type":"filebeat","version":"8.0.0"},"ecs":{"version":"1.6.0"},"host":"xxxxx","input":{"type":"stdin"},"log":{"file":{"path":""},"offset":0},"message":"bar"} {  false <nil>  4 0001-01-01 00:00:00 +0000 UTC 0s  map[] 0-0 } false}] dropped by rate_limit processor      {"instance_id": 1}
```

Reply:
Oh ok, nice then :+1: